Twitter.js

```diff
@@ -90,14 +90,17 @@ function scrape(doc, url) {
 		// e.g. 10:22 AM - 1 Feb 2018
 		var m = date.split('-');
 		if (m.length == 2) {
+			// times with AM
 			if (m[0].includes("AM")) m[0] = m[0].replace("AM", "").trim();
 			if (m[0].indexOf(":")==1) m[0] = "0" + m[0];
+			m[0].replace("12:", "00:");
+			// times with PM
 			if (m[0].includes("PM")) {
 				m[0] = m[0].replace("PM", "").replace(/\d+:/, function(matched) {
 					return (parseInt(matched)+12) + ":";
 				}).trim();
 			}
-			m[0].replace("24:", "00");
+			m[0].replace("24:", "12:");
```

Comment:
Do you still need this? You've already set 12: --> 00: in line 96 (which I first thought was a mistake, but it turns out to be quite elegant for this)

Reply:
A time with `12 PM` is replaced to `24 PM` by the addition of 12 in the lines above and thus have to be corrected again here. No?